src/pages/status/migration/index.jsx

```diff
@@ -16,6 +16,9 @@ const ORDERED = [
   ["bot-error", "Bot error"],
 ];
```

Comment:
What if we add a third field here, with their default state (True=shown, False=hidden).

Reply:
I thought about adding a third field here too.